resources/views/users/event.blade.php

```diff
@@ -0,0 +1,104 @@
+@extends('layouts.app')
+
+@section('title','Event')
+
```

Comment:
add a section here
```
@section('styles')
    <link rel="stylesheet" href="{{ mix('css/event.css') }}">
@endsection
```

Reply:
can you also do the same on `contact-us`, `faq` and `home` blade?

Reply:
I could chenge only my part(event blade).  Other parts are Aki-san's.  We will let Aki-san know tomorrow for the change.  Thank you.